e2e/specs/editForm.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { test } from "../core";
+import { expect } from "@playwright/test";
+import {
+  createForm,
+  createValueReference,
+  addFormResources,
+  deleteForm,
+} from "../commands/formOperations";
+import { FormBuilderPage } from "../pages";
+import { Form } from "../../src/types";
+
+let form: Form = null;
+test.beforeEach(async ({ api }) => {
+  form = await createForm(api);
+  const valueReference = await createValueReference(api);
+  await addFormResources(api, valueReference, form.uuid);
+});
+
+test("Should be able to edit a form", async ({ page }) => {
+  const formBuilderPage = new FormBuilderPage(page);
+
+  await formBuilderPage.gotoFormBuilder();
+  // Click on the edit schema button
+  await page.getByTestId(`editSchema${form.uuid}`).click();
```

Comment:
Can we do something like this?
```suggestion
  await formBuilderPage.editSchemaButton(`editSchema${form.uuid}`).click();
```

Reply:
We can but @jayasanka-sack and me had a discussion regarding these type of locators to keep it in the spec it self.  For example the toast selector. We cannot exactly catch it. We are using `getByText`. I think this is also similar use case. for that reason we can keep it here. WDYT @Piumal1999 @jayasanka-sack ?